scripts/install-windows-prereqs.ps1

```diff
@@ -588,28 +589,82 @@ function Install-Docker {
     Write-Output '[WARNING] Docker is no longer a prerequisite and installation thereof is deprecated.'
 }
 
-try {
+function Validate-SkipTasks {
+    Param(
+        [Parameter(Mandatory=$true)][array]$SkipTaskLower
+    )
+    $ValidSet = @("Nuget3", "Nuget6", "Python", "VisualStudio", "LLVM", "Git", "Shellcheck", "NSIS", "Docker", "OpenSSL", "VCRuntime")
+    foreach($item in $SkipTaskLower) {
+        if($ValidSet -notcontains $item) {
+            Write-Output "$item is not a valid package to skip."
+            return
+        }
+        Write-Output "Skipping $PackageToInstall"
+        return
+    }
+}
 
+try {
     if ($VerificationOnly) {
         Write-Output 'Verification only - Complete'
         Exit 0
     }
-
+    $lowercaseSkipTask = $SkipTask.ForEach({$_.ToLower()})
+    Validate-SkipTasks -SkipTaskLower $lowercaseSkipTask
     Start-LocalPackagesDownload
-
     Install-7Zip
-    Install-Nuget -Version 3
-    Install-Nuget -Version 6
-    Install-Python3
-    if (!$SkipVSInstall) {
+    if($lowercaseSkipTask -contains "nuget3") {
```

Comment:
An ordered hash table of each package to check each package would be more concise. But since one could get into a rabbit hole and we don't expect to add more packages, then this will suffice.

Suggestion: My opinion is this would be more readable by adopting a syntax that linters might not like... `if (x) { } else { }` in a single line.